manifests/vhost/wsgi.pp

```diff
@@ -0,0 +1,33 @@
+
+define apache::vhost::wsgi (
+  $wsgi_application_group                                                           = undef,
+  $wsgi_daemon_process                                                              = undef,
+  Optional[Hash] $wsgi_daemon_process_options                                       = undef,
+  $wsgi_import_script                                                               = undef,
+  Optional[Hash] $wsgi_import_script_options                                        = undef,
+  $wsgi_process_group                                                               = undef,
+  Optional[Enum['on', 'off', 'On', 'Off']] $wsgi_pass_authorization                 = undef,
+  $wsgi_chunked_request                                                             = undef,
+  $vhost                                                                            = $name,
+) {
+
+  if $wsgi_daemon_process {
```

Comment:
This was changed in https://github.com/puppetlabs/puppetlabs-apache/commit/eea8ecf63ee5ee56d9dc0af2664f834b7a79deb7 but here you might even unconditionally include it.